dlrover/python/tests/data/training.log

```diff
@@ -0,0 +1 @@
+error code is 507035
```

Comment:
Better intercept part of the original log as test data and simulate the actual scenario as much as possible.